.travis.yml

```diff
@@ -0,0 +1,62 @@
+sudo: required
+
+dist: xenial
+
+language: java
+
+services:
+  - docker
+
+env:
+  - JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+
+# Run before every job
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -y openjdk-8-jdk
+  - PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
+  - java -version
+  - uname -a
+  - chmod +x pom.xml
+
+# Only running report_coverage and deploy stage (after merge) to master branch
+stages:
+  - test
+  - name: report_coverage
+    if: branch = master AND type != pull_request
+  - name: deploy
+    if: branch = master AND type != pull_request
+
+# List of jobs to run, tied to specific stages
+jobs:
+  include:
+    - stage: test
+      name: unitAndFunctionalTests
+      script:
+        - mvn -Dsurefire.rerunFailingTestsCount=2 test -DskipITs -B
+    - stage: test
+      name: integrationTests
+      # To ensure docker containers are fully up and running we sleep 60s
+      before_script:
+        - source src/main/docker/env.bash
+        - docker-compose -f src/main/docker/docker-compose.yml up
+        - sleep 60
+      script:
+        - mvn verify -DskipUTs
+    - stage: deploy
+      name: deployGitHubPages
```

Comment:
Have we decided to use github pages for the gerrit plugin? Maybe we want a better solution? And is the token defined in the Travis environment? I suggest we remove this stage completely until we have reached a decision about using GitHub pages (and how)

Reply:
removed